.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,42 @@
+name: "Code Scanning - Action"
+
+on:
+  workflow_dispatch:
+  schedule:
+    #        ┌───────────── minute (0 - 59)
```

Comment:
Let's remove these they add some noise, and we have a few cron workflows without the comments so lose some consistency.